packages/bundler/src/utils/Constants.ts

```diff
@@ -125,3 +125,5 @@ export const UserOpWaitForTxHashMaxDurationIntervals: { [key in ChainId]?: numbe
   [ChainId.CHILIZ_MAINNET]: 20000,
   [ChainId.CHILIZ_TESTNET]: 20000,
 };
+
+export const DEFAULT_ENTRYPOINT_ADDRESS = "0x5ff137d4b0fdcd49dca30c7cf57e578a026d2789";
```

Comment:
could also import this from the baseValidationModule, but this is also fine if we don't wanna have circular dependency

Reply:
If it's fine like this I'm gonna let it